qutip/tests/core/data/test_norm.py

```diff
@@ -0,0 +1,73 @@
+from . import test_mathematics as testing
+import numpy as np
+import scipy as sc
```

Comment:
Can we avoid shortening `scipy`?  I don't really see any benefit to it, and it just makes things harder to read.  Shortening is ok for `numpy` because _everyone_ does that, but imo pretty much anything else should be spelled out in full in library code for readability.